### ISSUE TITLE
Build and test Parsey McParseface for proof of concept

### ISSUE THREAD

**Issue by cemunds** [Fallscout/ABSA-TensorFlow#2]
Link to Repo:
https://github.com/tensorflow/models/tree/master/syntaxnet

**Comment by DeastinY** [comment]
I tried setting up Syntaxnet on my arch machine for CUDA support, but did not manage to get it up and running in ~ 3 hours. Most issues are related to *bazel*. I can run Syntaxnet in a docker container, but did not yet manage to get CUDA working in there.

**Comment by cemunds** [comment]
I set up SyntaxNet on an Ubuntu VM and it works just fine. Did not fully understand the configuration yet, but I am able to annotate files. It outputs sentences like
"Why_WRB do_VBP Tesco_NNP fruit_NN squash_NN drinks_NNS have_VBP so_RB much_JJ salt_NN in_IN them_PRP ?_."
which I think we can definitely work with.

**Comment by jerryspan** [comment]
Syntaxnet had a major update today. See post here: https://research.googleblog.com/2017/03/an-upgrade-to-syntaxnet-new-models-and.html

**Comment by DeastinY** [comment]
> This upgrade incorporates nearly a year’s worth of our research on multilingual language understanding, and is available to anyone interested in building systems for processing and understanding text. 

> [...] the upgrade extends TensorFlow to allow joint modeling of multiple levels of linguistic structure, and to allow neural-network architectures to be created dynamically during processing of a sentence or document.

Introducing ParseySaurus
Character-based models are easier to build and improve analysis on some languages.

> The ParseySaurus models are far more accurate than Parsey’s Cousins (reducing errors by as much as 25%), particularly for morphologically-rich languages like Russian, or agglutinative languages like Turkish and Hungarian. 

Furthermore they are in on some conferencce (The SIGNLL Conference on Computational Natural Language Learning) to further improve techniques in the framework, but that's in the future.

**tl;dr:** Probably not that important for us, as the update is focussed on multilingual application.
